src/views/chats/Home/__tests__/HomeChatHeaders.spec.js

```diff
@@ -0,0 +1,131 @@
+import { mount } from '@vue/test-utils';
+import { beforeEach, describe } from 'vitest';
+import { createTestingPinia } from '@pinia/testing';
+
+import { useRooms } from '@/store/modules/chats/rooms';
+import { useDiscussions } from '@/store/modules/chats/discussions';
+
+import HomeChatHeaders from '../HomeChatHeaders.vue';
+
+const createWrapper = ({ store }) => {
+  return mount(HomeChatHeaders, { global: { plugins: [store] } });
+};
+
+describe('HomeChatHeaders.vue', () => {
+  let store;
+  let wrapper;
+
+  beforeEach(() => {
+    store = createTestingPinia({
+      initialState: {
+        rooms: {
+          activeRoom: null,
+        },
+        discussions: {
+          activeDiscussion: null,
+        },
+      },
+    });
+    wrapper = createWrapper({ store });
+  });
+
+  it('renders loading state correctly', async () => {
+    await wrapper.setProps({ isLoading: true });
+
+    expect(
+      wrapper.findComponent('[data-testid="chat-header-loading"]').exists(),
+    ).toBe(true);
+
+    expect(
+      wrapper.findComponent({ name: 'UnnnicChatsHeader' }).isVisible(),
+    ).toBe(false);
+  });
+
+  it('renders room header when room is active and discussion is inactive', async () => {
+    const roomStore = useRooms();
```

Comment:
It is not necessary to redo stores in each test, this can be in the scope of describe, which will be accessible in all tests